### PR TITLE
Shows an error-message to a user

### LINKED ISSUE #135: wrong-password message is missing
There must be a message that the username/password is not correct on the login mask. Also there must be a message if it is correct, but the user is inactive. Both messages must be generated by the auth-service and displayed in the client. The relevant strings are here: https://github.com/OpenSlides/OpenSlides/blob/master/server/openslides/users/views.py#L895

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
Fixes #135 

Waits for [openslides-auth-service#69](https://github.com/OpenSlides/openslides-auth-service/pull/69)...

### CODE REVIEW

File: client/src/app/site/login/components/login-mask/login-mask.component.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ export class LoginMaskComponent extends BaseComponent implements OnInit, OnDestr
             this.loginForm.setErrors({
                 notFound: true
             });
-            this.loginErrorMsg = e;
+            this.matSnackBar.open(e, 'Ok');
```
Comment (FinnStutzenstein) [inline client/src/app/site/login/components/login-mask/login-mask.component.ts:176]: this is weird. why do you open the snackbar? In OS3 no such snackbar opens, rather the error is shown below the password field.

Comment (GabrielInTheWorld) [reply client/src/app/site/login/components/login-mask/login-mask.component.ts:176]: Well, it is strange, that there is no error message shown. The code is still there...
That is, why I open a snackbar...

Comment (FinnStutzenstein) [reply client/src/app/site/login/components/login-mask/login-mask.component.ts:176]: Please find out, why nothing is shown there

Comment (GabrielInTheWorld) [reply client/src/app/site/login/components/login-mask/login-mask.component.ts]: Done.

File: client/src/app/site/login/components/login-mask/login-mask.component.ts [hunk 2]
```diff
@@ -171,9 +171,7 @@ export class LoginMaskComponent extends BaseComponent implements OnInit, OnDestr
             await this.authService.login(this.loginForm.value.username, this.loginForm.value.password);
         } catch (e) {
             // this.overlayService.hideSpinner();
-            this.loginForm.setErrors({
-                notFound: true
-            });
+            this.loginForm.get('password').setErrors({ notFound: true });
```
Comment (FinnStutzenstein) [inline client/src/app/site/login/components/login-mask/login-mask.component.ts:174]: Strange, I wonder why it works in OS3 :D

**Pull request merged**